lib/src/config_resolver.rs

```diff
@@ -116,6 +120,13 @@ fn matches_path_prefix(candidates: Option<&[PathBuf]>, actual: Option<&Path>) ->
     }
 }
 
+fn matches_platform(candidates: Option<&[String]>) -> bool {
+    candidates.is_none_or(|candidates| {
+        candidates.contains(&std::env::consts::FAMILY.to_owned())
+            || candidates.contains(&std::env::consts::OS.to_owned())
```

Comment:
nit: `candidates.iter().any(... || ...)` or `candidates.iter().contains() || ...` to remove unneeded allocation.